app/src/main/java/org/jellyfin/androidtv/auth/model/Server.kt

```diff
@@ -9,8 +11,13 @@ data class Server(
 	var id: UUID,
 	var name: String,
 	var address: String,
-	var dateLastAccessed: Date = Date(0)
+	val version: String? = null,
+	val loginDisclaimer: String? = null,
+	var dateLastAccessed: Date = Date(0),
 ) {
+	private val serverVersion = version?.let(ServerVersion::fromString)
+	val versionSupported = serverVersion != null && serverVersion >= ServerRepository.minimumServerVersion
```

Comment:
Should we consider `null` versions valid? At one point I ran into a case where the version hadn't been fetched yet so it said it was invalid, but retrying it worked as expected.

Reply:
The reason it didn't work that one time for you is because you already had server info stored from a previous development version. If you add a new server right now the app fetches the version when it stores it. It shouldn't be able to be null anymore.
The nullability in the model is for compatibility with older 0.12 builds. If I removed it the app would crash for users with their own builds.